packages/blitz/src/errors.ts

```diff
@@ -20,10 +20,13 @@ export class AuthenticationError extends Error {
   }
 }
 if (process.env.JEST_WORKER_ID === undefined) {
-  SuperJson.registerClass(AuthenticationError, {
-    identifier: "BlitzAuthenticationError",
-    allowProps: errorProps,
-  })
+  if (!(globalThis as any).ran) {
+    SuperJson.registerClass(AuthenticationError, {
+      identifier: "BlitzAuthenticationError",
+      allowProps: errorProps,
+    })
+    ;(globalThis as any).ran = true
```

Comment:
- did you consider any logic problems with using `.ran` for all classes?
- is there a better way to handle the globalThis type?